src/test/unit/model/indexing/lvalue_test.cpp

```diff
@@ -352,6 +352,7 @@ TEST(ModelIndexing, lvalueVecMulti) {
   EXPECT_FLOAT_EQ(ys(1), xs(3));
   EXPECT_FLOAT_EQ(ys(2), xs(4));
   test_throw(xs, index_list(index_min(0)), ys);
+  test_throw_ia(xs, index_list(index_min(1)), VectorXd::Ones(7));
```

Comment:
What does `index_min(x)` do?

Reply:
For vectors is does a slice from `min:N` and for matrices if slices on rows from `min:N` (or columns if it's the second index argument). There's a little helper doc at the top of `lvalue.hpp` and `rvalue.hpp` for what each index type does